webapp/channels/src/components/admin_console/message_export_settings.tsx

```diff
@@ -19,9 +19,11 @@ import JobsTable from './jobs';
 import SettingsGroup from './settings_group.jsx';
 import TextSetting from './text_setting';
 import RadioSetting from './radio_setting';
+import {AdminConfig} from "@mattermost/types/lib/config";
+import { DeepPartial } from 'redux';
 
 export default class MessageExportSettings extends AdminSettings {
```

Comment:
AdminSetting component is expecting to receive prop and state types. Please type them accordingly pass like this `AdminSettings<State, Props>`